open-data-day-tweets-2018/datapackage.json

```diff
@@ -0,0 +1,57 @@
+{
+  "profile": "data-package",
+  "resources": [
+    {
+      "name": "resource1",
+      "path": "data/subsetofopendatadaytweets.csv",
+      "schema": {
+        "fields": [
+          {
+            "name": "rowid",
+            "type": "integer",
+            "format": "default",
+            "description": "row id"
+          },
+          {
+            "name": "tweetid",
+            "type": "integer",
+            "format": "default",
+            "description": "tweet id"
+          },
+          {
+            "name": "retweetCount",
+            "type": "integer",
+            "format": "default",
+            "description": "number of retweets"
+          }
+        ]
+      },
+      "profile": "tabular-data-resource"
+    },
+    {
+      "name": "resource2",
+      "path": "scripts/opendataday.R",
+      "schema": {
+        "fields": []
+      }
+    },
+    {
+      "name": "resource3",
+      "path": "docs/geotagged_tweets.png",
+      "schema": {
+        "fields": []
+      }
+    }
+  ],
+  "keywords": [
+    "open data day",
+    "odd18"
+  ],
+  "name": "opendataday-tweets-2018",
+  "title": "Open Data Day Tweets",
+  "description": "A stripped-down version of open data day data mined from Twitter in March 2018. Data Brevity is as a result of  Twitter's Developer Policy.",
+  "homepage": "http://okfnlabs.org/blog/2018/03/08/open-data-day-tweets.html",
+  "version": "1",
+  "license": "Twitter Developer Policy",
+  "author": "Serah Rono <twitter.com/serahrono>"
```

Comment:
The usual pattern is `Foo Bar <foo@example.com> (https://example.com/)`. You can remove the e-mail if you want, so that becomes `Serah Rono (https://twitter.com/serahrono)`